### DISCUSSION TITLE
Is the project considered mantained?

### DISCUSSION THREAD

**Discussion by rapto** [stephenmcd/mezzanine#2074] [General]
There are several PR pending, which I noticed when I submitted one. Is the project mantained? I volunteer to take over it.

**Comment by Ryochan7** [comment]
There have been no commits to the repo since around November 2022. It really seems like the project is dead at this point. It is a shame as Mezzanine was a good CMS that offered real value over using vanilla Django. I have tried Django CMS and Wagtail (more recently) and I do not like either.

**Comment by molokov** [comment]
I am still using Mezzanine, even with latest Django (5.0.x) - I did have to make some fixes on my own fork (and thus am using that) but I don't know if anyone here is still maintaining this particular project.

**Reply by Ryochan7** [reply]
Thank you for updating Mezzanine. I am playing around with it right now. One issue that has cropped up with Django 5.0 is that the logout link no longer works. Logout can now only be done as a POST request.

**Reply by molokov** [reply]
See https://github.com/stephenmcd/mezzanine/discussions/2071 for a solution (updating your own template for the moment)

**Reply by Ryochan7** [reply]
thank you for the info

**Comment by rapto** [comment]
I (the company I am working for) am going to fork this and start using it.
I would be happy to cooperate if I were given permission in this repo.

**Comment by Parthian** [comment]
I hope it continues. Looked into using django_cms and wagtail a few years back but chose Mezzanine as I reckoned Mezzanine was the most Django like of the 3. Not sure how accurate that is but seemed like a good criteria in the end.
I've used it on 2 projects which needed simple blog/news home/about etc pages. Worked well for me.
